modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -40,6 +40,29 @@ To avoid disruption, you will need to repeat this process at the end of every su
 It runs locally to track installed products and subscriptions.
 Clients must be registered with the subscription manager to obtain certificates.
 
+Depending on where the {redhat} system is registered, the URL’s to which the system is authorized to replicate differ.
+
+{redhat} systems can be registered to three different infrastructures:
+
+1. {redhat} CDN (redhat.com)
+2. {redhat} Satellite Server
+3. RHUI (update infrastructure in the cloud)
+
+This guide focuses on a system registered to {redhat} CDN (redhat.com). Register at least one system there with an authorized subscription to obtain repository content.
+
+[IMPORTANT]
+====
+Entitlement certificates for RHUI (cloud-based systems) only authorize you to get content there.
+Satellite certificates for client systems require a Satellite server and subscription: such client certificates are UNSUPPORTED with {productname} server.
+====
```

Comment:
I feel like this admonition should be split into two: one about RHUI, one about Sat. Of course, then it means we have three admonitions in a row, which probably means it needs a little bit of re-thinking.

Reply:
@dvosburg ?

Reply:
I'm happy to edit this later on, anyway, so consider it non-blocking.